<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QDialog" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма за месяц</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_finalSum">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QCalendarWidget::SingleSelection</enum>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::ShortDayNames</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_counterDays">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>88</width>
     <height>192</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox_Monday">
      <property name="text">
       <string>Monday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_Tuesday">
      <property name="text">
       <string>Tuesday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_Wednesday">
      <property name="text">
       <string>Wednesday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_Thursday">
      <property name="text">
       <string>Thursday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_Friday">
      <property name="text">
       <string>Friday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_Saturday">
      <property name="text">
       <string>Saturday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_Sunday">
      <property name="text">
       <string>Sunday</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Стоимость занятия</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_sumInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
